### PR TITLE
Channel delete #340

### PR DESCRIPTION
**Pull request by elfkuzco**
An endpoint for deleting a channel.

### CODE REVIEW

**Comment by charles-co** [comment]
resolve conflicts

File: backend/channel_plugin/apps/channels/views.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from rest_framework.response import Response
 from rest_framework.views import APIView
 from rest_framework.decorators import api_view
-from .serializers import ChannelSerializer
+from .seriaizers import ChannelSeriali
```
Comment (charles-co) [inline backend/channel_plugin/apps/channels/views.py:5]: why?

Comment (elfkuzco) [reply backend/channel_plugin/apps/channels/views.py:5]: Sorry. Mistake while trying to resolve online.

**Review by elfkuzco — COMMENTED** [review]
Fixed bugs

**Comment by charles-co** [comment]
ready for PR review by mark

**Pull request merged**